app/views/spree/admin/payments/source_forms/_braintree.html.erb

```diff
@@ -0,0 +1,64 @@
+<fieldset data-id='credit-card' class="no-border-bottom">
```

Comment:
We *could* do that, but a simple first pass might be to just call the other _gateway view from this view, which would at least remove the duplication